CHANGES.md

```diff
@@ -15,6 +15,10 @@
   instances on Python 3.7+
   ([PR #351](https://github.com/cloudpipe/cloudpickle/pull/351))
 
+- Add support for pickling dynamic classes subclassing `typing.Generic`
```

Comment:
Aren't we doing more than this in this PR? Maybe you meant `GenericMeta`? I am not sure.

Reply:
Technically, it's subclasses of `Generic` as well as the special forms `Literal`, `Final`, `ClassVar`, but that's about it I think. That's all that was left over after the changes that were split off into the other PR.

I don't think there was an issue pickling instances of `GenericMeta` itself before, but I'd have to check.